src/subnet.rs

```diff
@@ -46,19 +27,30 @@ impl RouteSubnets {
 /// # Example
 ///
 /// ```
-/// use helium_config_service_cli::DevaddrRange;
+/// use helium_config_service_cli::DevaddrConstraint;
 /// use helium_config_service_cli::hex_field;
 /// use helium_config_service_cli::subnet;
 ///
 /// let start = hex_field::devaddr(0x11_22_33_40);
 /// let end = hex_field::devaddr(0x11_22_33_47);
-/// let range = DevaddrRange::new(start, end).unwrap();
+/// let range = DevaddrConstraint::new(start, end).unwrap();
 /// let subnet = range.to_subnet();
 ///
 /// let expected = vec!["11223340/29".to_string()];
 /// assert_eq!(subnet.subnets().unwrap(), expected);
 /// ```
-impl DevaddrRange {
+impl DevaddrConstraint {
+    pub fn new(start_addr: hex_field::HexDevAddr, end_addr: hex_field::HexDevAddr) -> Result<Self> {
+        if end_addr < start_addr {
```

Comment:
is `start_addr == end_addr` a valid case or should this be something like `if end_addr <= start_addr`?

Reply:
That is a valid configuration. 
It allows someone to select a single DevAddr to be routed.